### PR TITLE
translation: Add `Steward Guidelines` documentation in Hindi

### PR DESCRIPTION
**Pull request by EshaanAgg**
Changes:
- Added the Hindi translations for `Steward Guidelines`

 Screenshots of the change:
 NA

#### PR Checklist
- [X] `npm run lint` passes
- [X] [Inline documentation] is included / updated
- [X] [Unit tests] are included / updated

[Inline documentation]: https://github.com/processing/p5.js/blob/main/contributor_docs/inline_documentation.md
[Unit tests]: https://github.com/processing/p5.js/tree/main/contributor_docs#unit-tests

### CODE REVIEW

File: contributor_docs/hi/steward_guidelines.md [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+
+  - [उत्तर टेम्पलेट](./steward_guidelines.md#उत्तर-टेम्पलेट)
+  - [गिटहब सीएलआई](./steward_guidelines.md#गिटहब-सीएलआई)
+  - [सूचनाओं का प्रबंधन](./steward_guidelines.md#सूचनाओं-का-प्रबंधन)
+
+---
+
+## समस्याएँ
+
+हम अधिकांश स्रोत कोड योगदानों को एक समस्या के साथ शुरू करने की प्रोत्साहना करते हैं, और इस तरह, समस्याओं में अधिकांश चर्चाएँ होंगी। समस्या को समीक्षा करने के लिए लेने के चरण वास्तव में यह निर्भर करेगा कि यह कैसी समस्या है। रेपो विभिन्न प्रकार की समस्याओं को बेहतर ढंग से व्यवस्थित करने और समस्या लेखकों को अपनी समस्याओं के बारे में सभी प्रासंगिक जानकारी प्रदान करने के लिए [गिटहब समस्या टेम्पलेट](https://github.com/processing/p5.js/blob/main/.github/ISSUE_TEMPLATE) का उपयोग करता है। समस्या की समीक्षा की पहली कदम अक्सर भरे गए टेम्पलेट को देखना होगा और यह तय करना होगा कि क्या आपको अतिरिक्त जानकारी की आवश्यकता है (जैसे कि कुछ क्षेत्र भरे नहीं गए हों या गलत टेम्पलेट का प्रयोग किया गया हो)।
```
Comment (TanviKumar) [inline contributor_docs/hi/steward_guidelines.md:36]: और इस तरह, समस्याओं में अधिकांश चर्चाएँ होंगी - this isn't the meaning conveyed in the English docs, request you to change the same

**Review by TanviKumar — APPROVED** [review]

File: contributor_docs/hi/steward_guidelines.md (hunk 1 above)
Comment (EshaanAgg) [reply contributor_docs/hi/steward_guidelines.md]: Thanks for the review!

File: contributor_docs/hi/steward_guidelines.md [hunk 2]
```diff
@@ -0,0 +1,352 @@
+# स्टीवर्ड दिशानिर्देश
```
[the remark this answers is not in the document]
Comment (EshaanAgg) [reply contributor_docs/hi/steward_guidelines.md:1]: Thanks! Updated PR to `पुल अनुरोध` everywhere.

**Comment by TanviKumar** [comment]
@Qianqianye this PR can be merged

**Comment by Qianqianye** [comment]
Looks great, thanks @EshaanAgg @Divyansh013 @TanviKumar 💖
@all-contributors please add @EshaanAgg for translation

**Pull request merged**